src/validation.cpp

```diff
@@ -303,6 +303,7 @@ bool CheckSequenceLocks(const CTxMemPool& pool, const CTransaction& tx, int flag
                 }
             }
             lp->maxInputBlock = tip->GetAncestor(maxInputHeight);
+            assert(lp->maxInputBlock);
```

Comment:
If this assertion did not hold we wouldn't necessarily dereference any null pointer, would we?

Reply:
`tip->GetAncestor(maxInputHeight)` shouldn't return a nullptr because maxInputHeight is always less than the tip height, but did leave a comment explaining why we'd stop execution if that ever happened.